content/en/user-manual/assets/textures/texture-compression.md

```diff
@@ -59,6 +56,13 @@ If you would no longer want to use Basis, remove Basis compression from all text
 
 <img src="/images/user-manual/assets/textures/texture-compression/delete-basis-library.png" alt="Delete Basis Module" style="width: 400px;"/>
 
+## Basis Limitations
+
+There are some limitations of Basis texture compression in PlayCanvas.
+
+1. Basis can transcode to a number of formats natively supported by the GPU, namely DXT, ETC, ETC2, ASTC and PVR. The PVR format is quite limited, only supporting textures that have dimensions that are both square (same width and height) and power of two (e.g. 256, 512, 1024 and so on). Older iOS devices (with an A6 SoC or lower like the iPhone 5 and 5C) and older iOS versions (13.7 and lower) only support PVR. On these devices, a Basis texture that is non-square or non-power of two cannot be trancoded to PVR format but will instead be decompressed to a 16-bit 565 pixel format. So these textures will still display correctly although they may occupy more memory compared to the equivalent PVR texture.
```

Comment:
Move some of the information to line 11 so that this list is just about limitations rather than extra information.

>[Basis][1] is a 'super-compressed' texture format. It's a platform independent lossy block compression format that can be transcoded to the natively supported hardware compression format at runtime. Namely, DXT, ETC, ETC2, ASTC and PVR.

Reply:
Done